esp/src/lws.config.js

```diff
@@ -67,5 +67,6 @@ let rewrite = [
 
 module.exports = {
     port: 8080,
-    rewrite: rewrite
+    rewrite: rewrite,
+    stack: ['lws-basic-auth', 'lws-request-monitor', 'lws-log', 'lws-cors', 'lws-json', 'lws-compress', 'lws-rewrite', 'lws-blacklist', 'lws-conditional-get', 'lws-mime', 'lws-range', 'lws-spa', 'lws-static', 'lws-index']
```

Comment:
@GordonSmith this isn't strictly necessary for this PR. I realized the problem wasn't related to the "upload_" flag not being set for the ValidatePackage request, but rather that the body parser middleware of lws was having an issue with the size of the request. There's an existing issue in local-web-server (130), and the workaround recommended was to exclude the lws-body-parser middleware from the stack.